lib/gollum/templates/create.mustache

```diff
@@ -4,13 +4,15 @@
   <h1 class="py-4">Create New Page</h1>
 </div>
 <div id="wiki-content" class="create edit">
-	<div class="tabnav">
-	  <nav class="tabnav-tabs" aria-label="Foo bar">
-	    <a href="#" id="edit" class="tabnav-tab selected" aria-current="edit">Edit</a>
-	    <a href="#" id="preview" class="tabnav-tab" aria-current="preview">Preview</a>
-	  </nav>
-	</div>
-	
+  <div class="tabnav">
+    <nav class="tabnav-tabs" aria-label="Foo bar">
```

Comment:
While you're making edits here, could you maybe give the `aria-label` of the `nav` element a slightly better name than `Foo bar`? 😇

Reply:
Haha, of course. Not sure how this got by me!

Reply:
Resolved!